Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -256,6 +265,10 @@ - (void)setDelegate:(id<_WKWebAuthenticationPanelDelegate>)delegate
 #if ENABLE(WEB_AUTHN)
 static RetainPtr<NSArray> getAllLocalAuthenticatorCredentialsImpl(NSString *accessGroup, NSString *byRpId, NSData *byCredentialId)
 {
+    BOOL useAlternateKeychainAttribute = NO;
+#if USE(APPLE_INTERNAL_SDK)
+    useAlternateKeychainAttribute = shouldUseAlternateKeychainAttribute();
+#endif
```

Comment:
You repeat this block of code a half dozen times. I think it would be cleaner if you defined `shouldUseAlternateKeychainAttribute()` to do this work so you always just called that one function:

```
static BOOL shouldUseAlternateKeychainAttribute()
{
#if USE(APPLE_INTERNAL_SDK)
    if (![WebKit::getASCWebKitSPISupportClass() respondsToSelector:@selector(shouldUseAlternateKeychainAttribute)])
        return NO;

    return [WebKit::getASCWebKitSPISupportClass() shouldUseAlternateKeychainAttribute];
#else
    return NO;
#endif
}
```

Reply:
Done